Servatrice failing to compile on VS2017 after ccee0aa

As of https://github.com/Cockatrice/Cockatrice/commit/ccee0aa3fd7dfc77ef7ac246f8905cbf72ae8eba when attempting to compile servatrice using VS2017 it begins to fail with the following: 

```
5>------ Build started: Project: servatrice, Configuration: Release Win32 ------
5>version_string.cpp
5>MSVCRT.lib(exe_winmain.obj) : error LNK2019: unresolved external symbol _WinMain@16 referenced in function "int __cdecl __scrt_common_main_seh(void)" (?__scrt_common_main_seh@@YAHXZ)
5>C:\Cockatrice\build\servatrice\Release\servatrice.exe : fatal error LNK1120: 1 unresolved externals
5>Done building project "servatrice.vcxproj" -- FAILED.
========== Build: 4 succeeded, 1 failed, 2 up-to-date, 0 skipped ==========
```

Doing some quick searching it may have to do with some flags the possibly were removed with the commit.  I found a similar issue reported by some one else for another project at (https://stackoverflow.com/questions/39689162/qt-project-in-visual-studio-2015-unresolved-external-symbol-wwinmain)

Rolling back to the commit prior to this results in a successful compile.

--- a/servatrice/CMakeLists.txt
+++ b/servatrice/CMakeLists.txt
@@ -109,7 +109,11 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
 # Build servatrice binary and link it
 ADD_EXECUTABLE(servatrice MACOSX_BUNDLE ${servatrice_SOURCES} ${servatrice_RESOURCES_RCC} ${servatrice_MOC_SRCS})
 
-TARGET_LINK_LIBRARIES(servatrice cockatrice_common ${CMAKE_THREAD_LIBS_INIT})
+if(MSVC)
+    TARGET_LINK_LIBRARIES(servatrice cockatrice_common ${CMAKE_THREAD_LIBS_INIT} Qt5::WinMain)
+else()
+    TARGET_LINK_LIBRARIES(servatrice cockatrice_common ${CMAKE_THREAD_LIBS_INIT})
+endif()
 qt5_use_modules(servatrice ${SERVATRICE_LIBS})
 
 # install rules

